mono/mini/exceptions-amd64.c

```diff
@@ -151,6 +151,17 @@ LONG CALLBACK seh_vectored_exception_handler(EXCEPTION_POINTERS* ep)
 	switch (er->ExceptionCode) {
 	case EXCEPTION_STACK_OVERFLOW:
 		if (!mono_aot_only && restore_stack) {
+			if (er->NumberParameters == 2) {
```

Comment:
What is the significance of the value `2` here? Can you add a comment with some details about that?